src/components/data/QueryAuth0.js

```diff
@@ -114,7 +121,21 @@ export default class QueryAuth0 extends React.PureComponent {
         this.postProcessAuthResult(authResult);
       } catch (err) {
         window.clearTimeout(accessTokenRefreshLoopTimer);
-        this.props.authenticationFailed(err);
+        if (
+          typeof err === 'object' &&
+          (err.error === 'timeout' || err.error === 'login_required')
+        ) {
+          // Plain and simple, the silent authentication could not be done because the
+          // single-sign-in wants to revisit.
+          console.warn("checkSession didn't work.", err);
```

Comment:
my sentry PR will have all `console.warn` calls being recorded in sentry. ssince this error is largely "handled" we should either make this `console.debug` or `console.info` or ditch it altogether.